### PR TITLE
Postgis 2.2.1

### PR DESCRIPTION
**Pull request by kakoni**

### CODE REVIEW

File: Library/Formula/postgis.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Postgis < Formula
   desc "Adds support for geographic objects to PostgreSQL"
   homepage "http://postgis.net"
-  url "http://download.osgeo.org/postgis/source/postgis-2.2.0.tar.gz"
-  sha256 "66f0f8480d535959b8bb9abb5ee5d602d1001413ca770df120baf3de627f9e91"
+  url "http://download.osgeo.org/postgis/source/postgis-2.2.1.tar.gz"
+  sha256 "0fe500b0250203aac656bfa8f42f8458b63f33258404844e066e0e535988fa09"
   revision 1
```
Comment (xu-cheng) [inline Library/Formula/postgis.rb]: Please remove the revision line. Thanks.

Comment (kakoni) [inline Library/Formula/postgis.rb]: Done!

**Comment by UniqMartin** [comment]
Sorry, but some of these collisions like with the just updated `postgresql` are sometimes hard to avoid, if the updates are done by different contributors. Can you please rebase your commit(s) onto the most recent `master`? If you could also squash them to a single commit and make the commit message `postgis 2.2.1`, that would be awesome!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/de8b8ff7b9e37e070e802fe0184ef145cb8e7a7c. Thank you for your contribution to Homebrew @kakoni; we appreciate it! :smiley_cat:

**Pull request closed without merging**